### PR TITLE
Ticket manager

### PR DESCRIPTION
**Pull request by oliviaj0422**
Any other exceptions could be added? Please do not review Ticket, it's there for testing purposes.

### CODE REVIEW

**Comment by qiaoruzhang** [comment]
Very clean and good

**Review by Jasmine-Zhuang — COMMENTED** [review]
Nice, I agree with your name choice

File: src/main/java/Exceptions.java [hunk 1]
```diff
@@ -20,6 +20,10 @@
 
     public final static String INVALID_AMOUNT = "Please enter a valid integer between 1 and 1000000.";
     public final static String INSUFFICIENT_BALANCE = "Insufficient balance. Please load first.";
+    
+    // error messages associated with point redemption.
+    
+    public final static String INSUFFICIENT_POINTS = "Insufficient points to redeem."
```
Comment (Jasmine-Zhuang) [inline src/main/java/Exceptions.java]: Yes, with this, we can display this message in upper level class if customer want to redeem a rewards item but has insufficient points. Nice to add it.

Comment (Jasmine-Zhuang) [reply src/main/java/Exceptions.java]: BTW at your line 26, you forget to have ;

**Review by Jasmine-Zhuang — APPROVED** [review]

File: src/main/java/Exceptions.java (hunk 1 above)
Comment (oliviaj0422) [reply src/main/java/Exceptions.java]: Nice catch! Got it!

File: src/main/java/PHManager.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*
```
Comment (Jasmine-Zhuang) [inline src/main/java/PHManager.java:1]: typo at line 2 for " ticktes ", may change to tickets

**Comment by Jasmine-Zhuang** [comment]
When I update the ticket.java and ticketManager there is conflict in the manager line 68(the constructor)

**Pull request merged**